dataprovider/providers/aws/aws_data_provider_test.go

```diff
@@ -0,0 +1,120 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
not sure the file name should have `aws` prefix, as it's part of `aws` package already